test/e2e/network_test.go

```diff
@@ -290,8 +290,8 @@ func (tc *testContext) deleteNamespace(name string) error {
 	return tc.kubeclient.CoreV1().Namespaces().Delete(context.TODO(), name, metav1.DeleteOptions{})
 }
 
-// deployWindowsWebServer creates a deployment with a single Windows Server pod, listening on port 80
-func (tc *testContext) deployWindowsWebServer(name string, affinity *v1.Affinity) (*appsv1.Deployment, error) {
+// DeployWindowsWebServer creates a deployment with a single Windows Server pod, listening on port 80
```

Comment:
does this func have to be exported? Looks like the new call in `test/e2e/upgrade_test.go` is still from within the same package

Reply:
Correct! 
Thanks for catching that !